test/multi_tenancy.sh

```diff
@@ -74,51 +80,60 @@ function create_user_and_assign_to_projects() {
 }
 
 function add_message_to_index() {
-    # project is $1
-    # message is $2
-    # espod is $3
+
+    local namespace=$1
     local project_uuid=$( oc get project $1 -o jsonpath='{ .metadata.uid }' )
     local index="project.$1.$project_uuid.$(date -u +'%Y.%m.%d')"
-    os::log::debug $( curl_es_pod "$3" "/$index/multi-tenancy-test/" -XPOST -d '{"message":"'${2:-"multi-tenancy message"}'"}' | python -mjson.tool 2>&1 )
+    local pod=$3
+
+    local xff="-H X-Forwarded-For:127.0.0.1"
+    local xocpns='-H X-Ocp-Namespace:'"$namespace"
+    local contenttype="-H Content-Type:application/json"
+
+    local payload="{\"log\":\"log message 1\",\"stream\":\"stderr\",\"time\":\"2014-09-25T21:15:03.499185026Z\",\"kubernetes\":{\"namespace_name\":\"$namespace\",\"pod_name\":\"synthetic-logger-0.25lps-pod\"},\"docker\":{\"container_id\":\"container123\"}}"
+    local alias="{\"actions\":[{\"add\":{\"index\":\"$index\",\"alias\":\"app\"}}]}"
+
+    # add index
+    os::log::debug $( curl_es_pod "$pod" "/$index/multi-tenancy-test/" -XPOST $xff $xocpns $contenttype -d "$payload" | python -mjson.tool 2>&1 )
```

Comment:
Change the doc type here to `_doc` since doc types are going away in future releases of ES and '_doc' is the default one

Reply:
done